IPython/paths.py

```diff
@@ -9,8 +9,7 @@
 from IPython.utils.importstring import import_item
 from IPython.utils.path import (
     get_home_dir, get_xdg_dir, get_xdg_cache_dir, compress_user, _writable_dir,
-    ensure_dir_exists, fs_encoding, filefind
-)
+    ensure_dir_exists, fs_encoding)
```

Comment:
It is [documented](http://ipython.readthedocs.io/en/stable/api/generated/IPython.utils.path.html#IPython.utils.path.filefind) as part of `IPython.utils.path`, though, not this module. Do we know of people using it from here? I think that if we start worrying in general about things that are accessible because they're imported into that module, we'll go mad.